src/test/java/hw9/ValidateUtilTest.java

```diff
@@ -0,0 +1,40 @@
+package hw9;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ValidateUtilTest {
+
+
+    @Test
+    public void isValidPhoneNumber() {
+        assertEquals(true, ValidateUtil.isValidPhoneNumber("+380974929758"));
```

Comment:
In this case, you could use assertTrue(). It is more elegant.

Reply:
thanks, fixet to assertTrue()